ios/Exponent/Supporting/Info.plist

```diff
@@ -43,6 +43,14 @@
 				<string>host.exp.exponent</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.603386649315-vp4revvrcgrcjme51ebuhbkbspl048l9</string>
+			</array>
+		</dict>
```

Comment:
This will only work in standalone, should we put something in to prevent the use of this module in expo client? I imagine in `Expo.Google` we could just force the behavior to web like we have been doing. The issue is that the newer version of Google Sign-In requires a custom URI Scheme to be added.

Reply:
Rather than silently switching behavior to the web flow in the Expo client, I think we should throw a clear error saying that native login is supported only in standalone apps. This way it makes people aware of the difference and avoids differences in behavior between the Expo client and standalone apps -- we want to avoid discrepancies between dev and prod.

Reply:
true, this was mostly for internal testing.

Reply:
**Adding more context to this**
I actually added this to prevent a native crash that Google causes when this value isn't present. 😖